src/translate.py

```diff
@@ -1092,6 +1092,19 @@ def format(self, fmt: Formatter) -> str:
         if self.op in PSEUDO_FUNCTION_OPS:
             return f"{self.op}({lhs}, {rhs})"
 
+        # Write out flag checks such as '(var & literal) == 0' or '(var & literal) != 0' to be (var & literal) or !(var & literal)
+        if (
+            fmt.coding_style.brief_flag_checks
+            and isinstance(left_expr, BinaryOp)
+            and left_expr.op == "&"
+            and isinstance(right_expr, Literal)
+            and right_expr.value == 0
+        ):
+            if self.op == "==":
+                return f"!({strip_parens(lhs)})"
+            elif self.op == "!=":
+                return f"({strip_parens(lhs)})"
```

Comment:
This simplification is only valid within if/while, not for e.g. `x = ((a & 1) != 0)`. I think you want to move this logic to `simplify_condition`. (Where you also don't have to fiddle with formatting stuff like parentheses, nicely enough)